### ISSUE TITLE
Offline payment showing transaction QR

### ISSUE THREAD

**Issue by eloyesp** [dashpay/dash-wallet#732]
Bitcoin wallet adds an option to show the full transaction as a QR code, that can be scaned and broadcasted by the receiving phone. It makes sense when the merchant does have Internet available but the client does not.

**Comment by HashEngineering** [comment]
Yes, the code is still in the app, but showing the QR code is not exposed.
